source/fundamentals/authentication/mechanisms.txt

```diff
@@ -125,6 +125,40 @@ in the following sample code.
    SCRAM </release-notes/3.0-scram/>`, any ``MONGODB-CR`` user
    authentication requests fail.
 
+
+``MONGODB-AWS``
+---------------
+
+.. note::
+   The MONGODB-AWS authentication mechanism is only available in MongoDB
+   versions 4.4 and later.
+
+The ``MONGODB-AWS`` authentication mechanism uses your Amazon Web Services
+Identity and Access Management (AWS IAM) credentials to authenticate your
+user.
+
+To connect to a MongoDB instance with ``MONGODB-AWS`` authentication
+enabled, specify the ``MONGODB-AWS`` authentication mechanism and pass
+your ``AWS_ACCESS_KEY_ID`` and ``AWS_SECRET_ACCESS_KEY`` credentials to the
+driver when you attempt to connect. If your AWS login requires a session
+token, you must include your ``AWS_SESSION_TOKEN`` as well.
+
+The driver checks the following sources for your credentials in order:
+
+1. Connection string
+2. Environment variables
+3. AWS ECS endpoint specified in ``AWS_CONTAINER_CREDENTIALS_RELATIVE_URI``
+4. AWS EC2 endpoint. For more information see `IAM Roles for Tasks
```

Comment:
"For more information see `IAM Roles for tasks` ... "
should be:
"For more information, see `IAM Roles for tasks` ..."

(comma after information)